content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,92 @@
+## Installation
+
+- **Added** experimental [manifest and profile commands](docs/setup/install/operator/) to install and manage the Istio control plane for evaluation.
+
+## Traffic Management
+
+- **Added** automatic determination of protocol types. This removes the need for you to explicitly name service ports as `http` or `tcp`.
+- **Added** a mode to the Gateway API for mutual TLS operation. The new mode can read certificates via SDS.
+- **Added** Sidecar proxy readiness checks now check Envoy's readiness status.
+- **Fixed** issues present when a service communicates over the network first in permissive mutual TLS mode for protocols like MySQL and MongoDB.
+- **Improved** container ports are no longer required in the pod spec. All ports will be captured by default.
+- **Improved** the `EnvoyFilter` API. You can now add or update all configurations.
+- **Improved** the Redis load balancer to now default to [`MAGLEV`](https://www.envoyproxy.io/docs/envoy/v1.6.0/intro/arch_overview/load_balancing#maglev) when using the Redis proxy.
+- **Improved** load balancing to direct traffic to the [same region and zone](docs/ops/traffic-management/locality-load-balancing/) by default.
+- **Improved** Pilot performance. The reduction in CPU utilization could be up to 10 times depending on deployment.
+- **Improved** the `ServiceEntry` API to allow for the same hostname in different namespaces.
+- **Improved** the [Sidecar API](/docs/reference/config/networking/v1alpha3/sidecar/#OutboundTrafficPolicy) to customize the `OutboundTrafficPolicy` policy.
+
+## Security
+
+- **Added** a unified logging pattern to Citadel agent.
+- **Added** metrics to the Citadel agent to monitor SDS push.
+- **Improved** the ingress gateway to provide client CA certificates when using SDS.
+- **Added** trust domain validation for Envoy proxies using mutual TLS. Improved validation only applies to the `Sidecars` resource.
```

Comment:
```suggestion
- **Added** trust domain validation to Envoy proxies, when mutual TLS is used. By default, authentication will fail if the client and server side trust domains do not match.
- **Improved** validation only applies to the `Sidecars` resource.
```

Reply:
`- **Improved** validation only applies to the `Sidecars` resource.` what does this mean?  I don't think this is true at all @yangminzhu

Reply:
@myidpt please try to limit the amount of information you include in the note. The goal is to make the release notes easy to consume by readers of all experience levels. Instead of adding nuance to the release note, please provide a link to the document that has that information.

Reply:
@yangminzhu please response to @howardjohn 's request so we can get this comment resolved.

If you feel the current release notes are sufficient please click the `Resolve conversation` button.  TIA 👍